core/cmd/local_client_test.go

```diff
@@ -392,8 +406,12 @@ func TestClient_LogToDiskOptionDisablesAsExpected(t *testing.T) {
 			}
 			require.NoError(t, os.MkdirAll(cfg.Dir, os.FileMode(0700)))
 
-			cfg.New().Sync()
-			filepath := filepath.Join(cfg.Dir, "log.jsonl")
+			lggr, close := cfg.New()
+			defer close()
+
+			lggr.Debug("test")
```

Comment:
Is this necessary?

Reply:
yeah, the file won't exists until you write to it. So this specific test validates that case.